app/shared_features/set_card/expanded/SetDataRow.js

```diff
@@ -31,13 +31,16 @@ class SetDataRow extends PureComponent {
         }
 
         var dataStyle = this.props.item.removed ? styles.removedData : styles.data;
+
+        let averageVelocity = this.props.item.averageVelocity;
+        let validVelocity = isFinite(averageVelocity) && averageVelocity > 0 ? averageVelocity : 'Invalid' 
```

Comment:
- [ ] This only affects the dislpay of it, it doesn't actually set the FLAG in data to invalid. See isValidData in RepDataMap

Reply:
- [ ] YOU ARE STILL NOT UPDATING REPDATAMAP's ISVALIDDATA FUNCTION

Reply:
- [ ] I'd rather not do this, reason being that if people remember seeing over 10 m/s velocities and shit before, then I don't want it to show invalid now. I more want FUTURE ones to be shown as invalid. So, revert these